code/modules/mob/living/basic/vermin/mouse.dm

```diff
@@ -137,6 +141,14 @@
 	if(ishuman(entered) && stat == CONSCIOUS)
 		to_chat(entered, span_notice("[icon2html(src, entered)] Squeak!"))
 
+/// Called when a mouse is hand-fed some cheese, it will stop being afraid of humans
+/mob/living/basic/mouse/proc/tamed(mob/living/tamer, obj/item/food/cheese/cheese)
+	new /obj/effect/temp_visual/heart(loc)
+	visible_message(span_notice("[src] becomes accustomed to your presence."))
```

Comment:
Balloon alert? This isn't useful as archival information either since mice names are fairly useless

Reply:
As it already has the little heart animation I'll just remove the message entirely, a balloon alert on top of that wouldn't communicate any more information.